packages/launch/src/focused-launch/success/index.tsx

```diff
@@ -35,6 +35,20 @@ const Success: React.FunctionComponent = () => {
 		[]
 	);
 
+	// Save the post before displaying the action buttons and launch succes message
+	const [ isPostSaved, setIsPostSaved ] = React.useState( false );
+	const { savePost } = useDispatch( 'core/editor' );
+
+	React.useEffect( () => {
+		const asyncSavePost = async () => {
+			if ( ! isPostSaved ) {
+				await savePost();
+				setIsPostSaved( true );
+			}
+		};
+		asyncSavePost();
+	}, [ isPostSaved, savePost ] );
+
```

Comment:
Save the post when the Success view is shown — this avoids a bug (described in [this comment](https://github.com/Automattic/wp-calypso/pull/51001/files#r593165382))